src/Microsoft.AspNet.Mvc.Core/Formatters/HttpNotAcceptableOutputFormatter.cs

```diff
@@ -1,40 +0,0 @@
-// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Threading.Tasks;
-using Microsoft.AspNet.WebUtilities;
-using Microsoft.Net.Http.Headers;
-
-namespace Microsoft.AspNet.Mvc
-{
-    /// <summary>
-    /// A formatter which does not have a supported content type and selects itself if no content type is passed to it.
-    /// Sets the status code to 406 if selected.
-    /// </summary>
-    public class HttpNotAcceptableOutputFormatter : IOutputFormatter
```

Comment:
> Hmm, I do not see a reason to keep it. A user can pretty easily write a formatter like this.

I'm not really won over by this. Messing with conneg is non-trivial, and we provide plenty of things in the box that are simple to write.